app/controllers/api/v1/accounts/credentials_controller.rb

```diff
@@ -33,6 +33,7 @@ def user_settings_params
       'setting_default_privacy' => source_params.fetch(:privacy, @account.user.setting_default_privacy),
       'setting_default_sensitive' => source_params.fetch(:sensitive, @account.user.setting_default_sensitive),
       'setting_default_language' => source_params.fetch(:language, @account.user.setting_default_language),
+      'setting_default_federation' => source_params.fetch(:sensitive, @account.user.setting_default_federation),
```

Comment:
This should fetch `:federation`, not `:sensitive`, I had to fix this in my branch.

Reply:
Regardless of what I think about this change, this setting is strange and hard to understand.

Is there a possibility to have this feature turned off for the instance (hopefully by default)?